deploy/debezium-connector.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: kafka-connectors
+objects:
+- apiVersion: kafka.strimzi.io/v1beta2
+  kind: KafkaConnector
+  metadata:
+    name: kessel-inventory-source-connector
+    labels:
+      strimzi.io/cluster: ${KAFKA_CONNECT_INSTANCE}
+  spec:
+    class: io.debezium.connector.postgresql.PostgresConnector
+    tasksMax: ${{MAX_TASKS}}
+    config:
+      database.server.name: ${DB_SERVERNAME}
+      database.dbname: ${DB_NAME}
+      database.hostname: ${DB_HOSTNAME}
+      database.port: ${DB_PORT}
+      database.user: ${DB_USER}
+      database.password: ${DB_PASSWORD}
+      topic.prefix: kessel-inventory
```

Comment:
Does this config param set the prefix for topics that debezium publishes to? Just trying to understand where this gets applied.